pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -315,11 +307,12 @@ effectiveStdenv.mkDerivation {
     ++ lib.optional enableFfmpeg ffmpeg
     ++ lib.optionals (enableFfmpeg && effectiveStdenv.isDarwin)
       [ VideoDecodeAcceleration bzip2 ]
-    ++ lib.optionals enableGStreamer (with gst_all_1; [ gstreamer gst-plugins-base gst-plugins-good ])
+    ++ lib.optionals (enableGStreamer && effectiveStdenv.isLinux) (with gst_all_1; [ gstreamer gst-plugins-base gst-plugins-good elfutils orc libunwind zstd ])
     ++ lib.optional enableOvis ogre
     ++ lib.optional enableGPhoto2 libgphoto2
     ++ lib.optional enableDC1394 libdc1394
     ++ lib.optional enableEigen eigen
+    ++ lib.optional enableVA libva
```

Comment:
Nit: I feel like changing from `optional` to `optionals` whenever a line is to be touched anyway

Reply:
I did a thing.

Reply:
The radical approach. I'm happy